backend/quadcore/restful/github/logout_github.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask, redirect, url_for, session, request, jsonify
+from flask_restful import Resource, Api
+from quadcore.manager.auth import AuthManager as am
+
+class LogoutGithub(Resource):
+
+    def get(self):
+        return {
+            "result": 1,
+            "reason": "You approached wrong way!"
+        }
+
+    def post(self):
+        session.pop('github_token', None)
+        return 'OK'
```

Comment:
Returning string is not recommended. Try to package data to Python dictionary, which will reformat to JSON.